### DISCUSSION TITLE
[fplus] Is it needed to change port name from fplus to FunctionalPlus

### DISCUSSION THREAD

**Discussion by miyanyan** [microsoft/vcpkg#38213] [Q&A] [answered]
### Is your feature request related to a problem? Please describe.

I find that `fplus` is mainly used in source code, and `FunctionalPlus` seems to be more clear :
* in url, github.com/Dobiasd/**FunctionalPlus**
* in readme, **FunctionalPlus** is a small header-only library supporting you in reducing code noise and in dealing with...
* in cmake, find_package(**FunctionalPlus** CONFIG REQUIRED)

### Proposed solution

rename this port

### Describe alternatives you've considered

/

**Comment by LilyWangLL** [comment]
If the name of this port is modified, it will affect the local cache of the port that has already been installed, and there is a port `frugally-deep` in vcpkg that depends on this port. 
However, on website https://repology.org/projects/?search=FunctionalPlus, the name of this port is `functionalplus`.

I will convert this issue to discussion, waiting for more suggestions.

**Reply by Osyotr** [reply]
What else is here to discuss? Pretty much the same story as `proj4` - make it alias for new port `functionalplus`, deprecate `fplus` and eventually remove it.

**Comment by vipcxj** [comment]
The port name seem to have to be all lowercase. I customized a port with a capitalized name, and when I named it as is, I got an error and had to change the name to all lowercase. This results in 2 related folders in the share folder, which doesn't affect usage, but it's not perfect!

**Reply by dg0yt** [reply]
> This results in 2 related folders in the share folder, which doesn't affect usage.

This isn't necessary in most cases. At least it is not necessary for CMake config.

**Comment by LilyWangLL** [comment, marked as answer]
Will be fixed by PR https://github.com/microsoft/vcpkg/pull/38346.
